### ISSUE TITLE
Edit menu shortcuts inconsistent / incorrect

### ISSUE THREAD

**Issue by rohrlich** [goki/gide#42]
From 2 machines
<img width="230" alt="screen shot 2018-10-19 at 6 14 06 pm" src="https://user-images.githubusercontent.com/22350787/47248912-1b9e9680-d3cb-11e8-8f37-d022f2801034.png">
![screen shot 2018-10-19 at 6 15 27 pm](https://user-images.githubusercontent.com/22350787/47248917-26f1c200-d3cb-11e8-89cb-36d138df365e.png)

**Comment by rohrlich** [comment]
Paste history is nice!

**Comment by rcoreilly** [comment]
:)  did you figure out what caused the mixup in shortcuts??

**Comment by rohrlich** [comment]
No, I through away my gide prefs and that didn't fully fix it. I can look into in the morning.
